test/functional/toolchains/google/test_bazel.py

```diff
@@ -49,8 +49,27 @@ def test_basic_exe(bazelrc, build_type, base_profile):
         assert "myapp/1.0: Hello World Debug!" in client.out
 
 
+@pytest.mark.parametrize("build_type", ["Debug", "Release", "RelWithDebInfo", "MinSizeRel"])
+@pytest.mark.tool("bazel", "7.1.2")
+@pytest.mark.skipif(platform.system() != "Darwin", reason="Only Darwin at the moment")
```

Comment:
Why if bazel has been defined in ``conftest.py`` for all platforms?

Reply:
You're completely right. It was a typo 😁